src/components/App/App.js

```diff
@@ -0,0 +1,18 @@
+import styled from 'styled-components/macro'
+import Countdown from '../Countdown/Countdown'
+
+function App() {
+  return (
+    <AppGrid>
+      <Countdown startMinutes={25} startSeconds={0} />
```

Comment:
are the props here necessaray? default values are 25min & 0sec, right?

Reply:
I see it exactly the same way. Since you don't have a config file that might store a lot of static values, you can leave out the props here. The app component does not need to know about this... 🚀

Reply:
True, thanks!